Profile view

jwanggggg opened the pull request:
Added a basic profile page that displays user's info.

mirahmed39 commented:
Hey Jonathan,
I have taken a look the PR and here are things I want to point out:

+ This view should be an authenticated view. Meaning before you render this page you have to make sure that a user is logged in. Right now I can type `localhost:3000/profile` and can access that page. You  can take a look at one of my routes on how to make it authenticated view.
+  It is better if you show the content inside a form and make the input read-only since we are not gonna bother allowing user to change their info for now. It looks much better and professional.
+  Instead of slapping the entire date format on the page, you can format it nicely to show only the necessary parts.
+ In your profile.js page, instead of passing all the properties of `req.user` separately in your render function, you can just pass `{user: req.user}` and in the .hbs file you can use `{{ user.firstName}}` etc.

Hope you find these helpful. If you have any confusions, don't forget to ping me on slack.

Sincerely,
Mir.

jwanggggg commented:
@mirahmed39, fixed the 4 changes that you mentioned. If you could take a look and see if what's you're looking for.

I could not find a clean way to format the date, so I split it into an array and used the individual strings. It looks aesthetically fine, but let me know if you know of a cleaner way to format the Date in a readable fashion.

mirahmed39 commented:
The date was not displaying at all in the page. I have removed some undefined properties on the user object such as user.weekRegistered etc and formatted the date to display only the month and year. Everything is working as expected. Good work!

Sincerely,
Mir.
